src/argilla/labeling/text_classification/rule.py

```diff
@@ -74,6 +74,31 @@ def author(self):
         """Who authored the rule."""
         return self._author
 
+    def update_label(self, label):
+        """Updates the label"""
+        self._label = label
+
+    def add_to_dataset(self, dataset: str):
+        """Add to rule to the given dataset"""
+        labeling_rule = LabelingRule(
+            query=self.query, label=self.label, labels=[], author=self.author or "None"
```

Comment:
As far as I remember, the author will be filled with the user request if no provided. We should remove de "None" defaults